### ISSUE TITLE
LIBC Incompatible

### ISSUE THREAD

**Issue by Alukardz** [ibm-messaging/mq-golang#207]
Hello,

When I run the go app, I get: 
/usr/lib64/libc.so.6: version `GLIBC_2.32' not found 
/usr/lib64/libc.so.6: version `GLIBC_2.34' not found

Is there any way to run the go binary on a machine that only has the libc version 2.28 with no way to upgrade, but already has the ibm libraries?

**Comment by ibmmqmet** [comment]
What do you mean by "the go app" and "the go binary"? This repo doesn't provide any binaries. Just the source code which you compile into your app.  Or someone has given you a pre-compiled app, in which case you need to ask them.

The only relevant binary involved here will be the MQ C client libraries whose prereq cannot be changed. Though I think it is much older than 2.32.

Or maybe you are talking about the actual go compiler - again nothing that can be done by asking here.

**Comment by Alukardz** [comment]
Yes that's the problem, the MQ C client libraries required are at least GLIBC_2.32, but the pc in question only has the 2.28, is there a version of mq-golang that can run on 2.28? or any other help?
